scripts/run_no_explanation.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+pretrained_model_path="microsoft/Phi-3-mini-128k-instruct"
+
+#data_path="data/MedQA-USMLE-4-options-test.csv" #medqa
+data_path="data/afri_med_qa_10k_v1_1_phase_1.csv" #afrimed-qa
+
+question_types=("mcq")
+prompt_type=base
+explanation=_no_exp
```

Comment:
this seems a bit odd since it's meant to be a boolean option or command line argument.